README.md

```diff
@@ -84,8 +84,9 @@ TEAMMATES team wishes to thank the following invaluable contributions.
 * [**School of Computing, National University of Singapore (NUS)**](http://www.comp.nus.edu.sg), for providing us with the infrastructure support to run the project.
 * [**Centre for Development of Teaching and Learning (CDTL)**](http://www.cdtl.nus.edu.sg/) of NUS, for supporting us with several *Teaching Enhancement Grants* over the years.
 * **Learning Innovation Fund-Technology (LIF-T)** initiative of NUS, for funding us for the 2015-2018 period.
-* **Google Summer of Code** Program, for including TEAMMATES as a mentor organization in *GSoC2014* and *GSoC2015* editions.
+* **Google Summer of Code** Program, for including TEAMMATES as a mentor organization in *GSoC2014*, *GSoC2015* and *GSoC2016* editions.
 * [**YourKit LLC**](ttps://www.yourkit.com), for providing us with free licenses for the [YourKit Java Profiler <img src="https://www.yourkit.com/images/yklogo.png" width='70'>](https://www.yourkit.com/.net/profiler/index.jsp) (an industry leading profiler tool for Java applications).
+* **Facebook Open Academy** Program, for including TEAMMATES as a mentor organization in FBOA 2016.
```

Comment:
Can move this item to right after GSoC? The idea is so that it's as identical as possible to the about page (`about.jsp`).